### PR TITLE
Loot & Loadout Rebalance

### PR DESCRIPTION
**Pull request by ProbablyCarl**
- - -
Completely overhauls loot drops within dungeons and elsewhere, returning rarity to firearms and encouraging melee.

Additionally, this removes many of the absurd BoS loadouts and retains the appropriate alternatives. Begone, every second man with a minigun! Woe be upon ye!
Atop that, it removes the ability for the BoS to mass produce plasma pistols.
- - -

### CODE REVIEW

File: code/modules/jobs/job_types/bos.dm [hunk 1]
```diff
@@ -470,7 +425,7 @@ Star Paladin
 /datum/outfit/loadout/spaladinc
 	name = "Senior Frontline Paladin"
 	backpack_contents = list(
-		/obj/item/gun/energy/laser/aer12 = 1,
+		/obj/item/gun/energy/laser/aer9 = 1,
 		/obj/item/gun/energy/laser/pistol=1,
```
Comment (Spc-Dragonfruits) [inline code/modules/jobs/job_types/bos.dm:429]:
```suggestion
		/obj/item/gun/energy/laser/pistol=1,
```
Suggest replacing this with an upgraded wattz 1k

**Review by Spc-Dragonfruits — CHANGES_REQUESTED** [review]
Generally okay save for the nitpick. I think it's plausible to give knights the choice between an AER9 and a Verminkiller.

File: code/modules/jobs/job_types/bos.dm (hunk 1 above)
Comment (ProbablyCarl) [reply code/modules/jobs/job_types/bos.dm:429]: Wattz is still the most powerful laser rifle hands down. Would rather not.
If it doesn't work out, we'll up it back to an AER12 or, perhaps, yes, a Wattz.

Comment (ThatGuyFromThatPlace) [reply code/modules/jobs/job_types/bos.dm:429]: > Wattz is still the most powerful laser rifle hands down. Would rather not. If it doesn't work out, we'll up it back to an AER12 or, perhaps, yes, a Wattz.

Watts 1k is the pistol version

**Comment by UntoldTactics** [comment]
Today I will remove the majority of BOS loadouts without changing the loadouts of any other faction, effectively reducing them to a state where they cannot contend with anyone, because their starting gear is dogshit as fuck. Amazing. Why?

**Comment by DiscordWizard** [comment]
I understand not wanting Paladins to have miniguns after the bullet changes but...

The Castellan should keep their weapons considering how other faction leaders' loadouts are, and nerfing literally the rest of the brotherhood is completely unnecessary. We can't even replenish our armory with wasteland gear because all energy weapon spawns were made rarer and some around the map (Which after the change would only spawn a laser pistol for some god forsaken reason) removed altogether. 

There is no reason for the BoS to bunker dive if they can't recover anything worthwhile for their mission, nor any reason for them to trade with other factions because... Well, what will they have to trade if energy weapon spawns are culled all over the map? And why would they sell the better guns to the faction that is now so gimped they can't enforce anything?

In short, this RP has made the BoS, the supposedly more well equipped faction of the normal 3, the worst off, and removed altogether most if not all of its options against the Enclave, who get a massive boon due to the minigun bullet buff. THEN it has removed most reasons for them to trade and not be a random band of (now underequipped) raiders desperate for a weapon that's not a shitty AER9. Mechanically, they suck. RP-wise, they are pretty much irrelevant.
